<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VarTabClass</class>
 <widget class="QWidget" name="VarTabClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>76</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="DeleteBranch">
   <property name="geometry">
    <rect>
     <x>439</x>
     <y>41</y>
     <width>90</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>90</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>90</width>
     <height>25</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Удалить ветвь</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Yes">
   <property name="geometry">
    <rect>
     <x>534</x>
     <y>40</y>
     <width>35</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>35</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>35</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string>Да</string>
   </property>
  </widget>
  <widget class="QPushButton" name="No">
   <property name="geometry">
    <rect>
     <x>575</x>
     <y>40</y>
     <width>35</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>35</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>35</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string>Нет</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="NumberOfOutcomes">
   <property name="geometry">
    <rect>
     <x>388</x>
     <y>41</y>
     <width>45</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>45</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>45</width>
     <height>25</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>8</number>
   </property>
  </widget>
  <widget class="QLabel" name="LabelOutcomes">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>41</y>
     <width>371</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>371</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>371</width>
     <height>25</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Количество исходов при выборе этого варианта:</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>452</x>
     <y>40</y>
     <width>76</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>76</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>76</width>
     <height>25</height>
    </size>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>582</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="LabelText">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Текст кнопки:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="ButtonText">
      <property name="minimumSize">
       <size>
        <width>470</width>
        <height>22</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="cursorPosition">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
